packages/core/src/tracing/instrumentation/protocols/graphql.js

```diff
@@ -98,7 +98,7 @@ function traceQueryOrMutation(
   operationDefinition,
   operationName
 ) {
-  const activeEntrySpan = cls.getCurrentSpan();
+  const activeEntrySpan = cls.getCurrentSpan(true);
```

Comment:
@basti1302 also related to the graphql change from your comment https://github.com/instana/nodejs/pull/416#discussion_r760925276.
There is also the same change at line 168.